nuspecs/Microsoft.AspNet.SignalR.ServiceBus3.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.AspNet.SignalR.ServiceBus3</id>
```

Comment:
Because we upgrade the dependency from 2 to 3 which has API breaking changes. We can talk about versioning with @muratg and @DamianEdwards offline. The most I care about in this PR is actually code changes.